### ISSUE TITLE
 Clarification on Fine-Tuning with real images

### ISSUE THREAD

**Issue by NasrinPrenuvo** [BBillot/SynthSeg#77]
Impressive job! I have a question regarding fine-tuning. Perhaps I haven't grasped the paper's content entirely. The paper suggests using synthesized images rather than real images during training, emphasizing the desire to enable the model to learn robust features rather than intensity. However, I'm confused about the purpose of fine-tuning my dataset. If I can't use real images during this process, wouldn't the model still prioritize intensity features over other crucial features? I'm a bit confused and would appreciate some clarification.

**Comment by BBillot** [comment]
you can definitely fine-tune on your dataset, but this will probably make the network forget robustness to different contrast/resolution. 

"If I can't use real images during this process, wouldn't the model still prioritize intensity features over other crucial features?"
-> intensity feature are very broad and can refer to a lot of things. Here we're teaching the network to disregard the contrast (i.e., the colour) of regions, but we're absolutely not teaching it to disregard intensities ! It's hard to understand what's going on in the network, but my guess is that the network learns to detect edges (which are intensity-based features) between structures so it can place boundaries. I know that training on purely synthetic (and unrealistic) data is counter-intuitive, but it gives you this very nice generalisation. SynthSeg will perform slightly worse than supervised networks tested on images similar to their training data, but these supervosed networks will start degrading as soon as you start changing the domain.

Sorry for the long answer, I hope it clarifies things a bit :)

**Comment by NasrinPrenuvo** [comment]
Thank you very much:)
So, I cannot achieve both the accuracy of supervised training and robustness to contrast and resolution simultaneously. I appreciate the clarification.

**Comment by NasrinPrenuvo** [comment]
Thanks for your reply!
Do you think it's helpful to fine-tune training on my dataset using only the labels of my images, not real images?

**Comment by BBillot** [comment]
Sure, you can try and see if it performs better :)
